__tests__/app.test.js

```diff
@@ -12,6 +12,17 @@ afterAll(() => {
   db.end();
 });
 
+describe("test", () => {
```

Comment:
Maybe a more descriptive name would be helpful to know what this test is doing when i run npm t and see the test names in the console